### PR TITLE
Update daily upgrade notifier message for non-npm installations

### PR DESCRIPTION
**Pull request by pdcastro**
Resolves: #842
Change-type: patch
Signed-off-by: Paulo Castro <paulo@balena.io>

### CODE REVIEW

File: lib/utils/update.ts [hunk 1]
```diff
@@ -25,26 +25,26 @@ const balenaUpdateInterval = 1000 * 60 * 60 * 24 * 1;
...
-	if (notifier.update != null) {
-		console.log(
-			'Notice that you might need administrator privileges depending on your setup\n',
-		);
+	const up = notifier.update;
+	if (up) {
+		notifier.notify({
+			defer: false,
+			message: `Update available ${up.current} → ${up.latest}\n
+https://github.com/balena-io/balena-cli/blob/master/INSTALL.md`,
```
Comment (pdcastro) [inline lib/utils/update.ts]: What URL do we want to use? -
1. https://github.com/balena-io/balena-cli/blob/master/INSTALL.md
2. https://www.balena.io/docs/reference/cli/
3. https://www.balena.io/docs/reference/cli/#installation
4. A tinyurl to any of the above, e.g. https://tinyurl.com/yymwhcaj

Option 3 would be good if it wasn't for the fact that the anchor link results in the **Installation** section title to get under the page header, leaving the **Getting Started** section prominent. Users would need some scrolling to figure it out, then click on a link to takes them to GitHub. All things considered, I think my preference is for option 1, without a tinyurl link. The CLI is usually used on large enough screens to accommodate the long URL, and then it's straight to the point and clear that the URL will land on INSTALL.md on GitHub.

Comment (hedss) [reply lib/utils/update.ts]: Concur.

**Comment by pdcastro** [comment]
force push = rebase on HEAD

**Comment by pdcastro** [comment]
@balena-ci retest

**Comment by pdcastro** [comment]
@balena-ci retest

**Pull request merged**